packages/docs-theme/src/styles/_examples.scss

```diff
@@ -75,6 +75,7 @@ $dark-options-background-color: $dark-gray5;
 
 .docs-example-options {
   flex: 0 0 auto;
+  margin-left: $pt-grid-size;
```

Comment:
move this to line 36 inside the `-row` block and you can remove `margin-left: 0` on line 49